ports/libsimpleservo/api/src/lib.rs

```diff
@@ -159,6 +158,7 @@ pub fn init(
     gl: Rc<dyn gl::Gl>,
     waker: Box<dyn EventLoopWaker>,
     callbacks: Box<dyn HostTrait>,
+    device_pixels_per_px: Option<f32>,
```

Comment:
This will require updating https://github.com/servo/servo/blob/6a54d600431621bfd4c4d4e49e7816e95deec68e/ports/libmlservo/src/lib.rs#L193 and https://github.com/servo/servo/blob/6a54d600431621bfd4c4d4e49e7816e95deec68e/ports/libsimpleservo/jniapi/src/lib.rs#L117 and https://github.com/servo/servo/blob/6a54d600431621bfd4c4d4e49e7816e95deec68e/ports/libsimpleservo/capi/src/lib.rs#L320.

However, I just noticed https://github.com/servo/servo/blob/6a54d600431621bfd4c4d4e49e7816e95deec68e/ports/libsimpleservo/api/src/lib.rs#L49 which is the value that is consulted when device_pixels_per_px is None in [this code](https://github.com/servo/servo/blob/master/components/compositing/compositor.rs#L1083-L1091). So I think we don't actually need to add this new argument; the device_pixels_per_px value is an override that is only meaningful in the glutin port, and the other ports already have a supported API for reporting the DPI factor, so we can revert this change to the `init` method.

Reply:
Thanks for catching this. I removed this change and pushed the rebased commit without this change.